src/pages/guide/comparison-with-leading-frameworks.ejs

```diff
@@ -141,7 +141,7 @@ const data = {
         <td>
           <span class="comp-yes">✅</span> <br>
           <p>Mobile web is just as easy to setup and supports everything from Desktop web.
-            <br><a href="#">Get Started</a>
+            <br><a href="ttps://nightwatchjs.org/guide/mobile-app-testing/introduction.html">Get Started</a>
```

Comment:
Please check the href attribute.

Reply:
Sorry, for underlooking and pushing these silly changes, Actually these changes are from my old PR. So, I simply copied from that PR, without testing them again. 
I will not repeat this again.